ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -422,13 +434,8 @@ def setUp(self):
         self.generate_arp_ping_packet()
 
         if self.reboot_type == 'warm-reboot':
-            # get the number of members down for sad path
             if self.preboot_oper:
-                if ':' in self.preboot_oper:
-                    oper_type, cnt = self.preboot_oper.split(':')
-                else:
-                    oper_type, cnt = self.preboot_oper, 1
-                self.log("Preboot Oper: %s Number down: %s" % (oper_type, cnt))
+                self.log(self.get_preboot_info())
```

Comment:
I'd suggest to put all the checks to the method `self.get_preboot_info()`.
It will document the code better(we will be know that we need to extract the info iff we are in WR).
```
if self.reboot_type == 'warm-reboot':
   if self.preboot_oper:
```

Reply:
done